src/web/docusaurus/docs/architecture.md

```diff
@@ -25,22 +25,6 @@ The following gives an overview of the current (i.e. 2.4.0) design of Telescope.
 - autodeployment server
 - test, build, and other automation tooling
 
-### Legacy Monolithic Back-end (1.0)
```

Comment:
Should we keep some of this to keep some history? Maybe in the "Initial Design" section.

Reply:
I suggest we keep it. When we reach a 4.0 release, we might want to restructure the docs again and decide whether we want to remove or keep some part of it.